addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6870,6 +6928,10 @@ registry.ImageTools = ImageHandlerOption.extend({
             const colors = img.dataset.shapeColors.split(';');
             return colors[parseInt(params.colorId)];
         }
+        if (widgetName === 'shape_anim_opt') {
+            const shapeImgWidget = this._requestUserValueWidgets('shape_img_opt')[0];
+            return shapeImgWidget && shapeImgWidget.getMethodsParams().animated;
+        }
```

Comment:
you should be able to use `_isAnimatedShape()` here

Reply:
Yes, since your commit. Thanks